modules/tinymce/src/plugins/lists/test/ts/browser/OutdentTest.ts

```diff
@@ -26,9 +39,9 @@ UnitTest.asynctest('browser.tinymce.plugins.lists.OutdentTest', (success, failur
 
     editor.focus();
```

Comment:
Some more focus calls that happen on each test, so these should be moved to a `beforeEach` hook.

- [ ] Move focus calls to `beforeEach` hook